src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/EmployerAccountControllerTests/CreateAccount/Given_Cookie_Data_Is_Null/WhenICreateAnAccount.cs

```diff
@@ -81,8 +79,7 @@ public void Arrange()
                 _userViewTestingService.Object,
                 logger.Object,
                 _flashMessage.Object,
```

Comment:
There is still a reference to the feature toggle in the mocked authorization service; that can be completely removed now as it was only necessary for the feature toggle on and off scenarios.

Reply:
Removed it